### PR TITLE
Add link to Scylla Source Available License (SSAL)

### PR DESCRIPTION
**Pull request by tzach**

### CODE REVIEW

**Comment by tzach** [comment]
@mmatczuk FYI

**Comment by mmatczuk** [comment]
What about our [proprietary license](https://github.com/scylladb/scylla-manager/blob/master/license/LICENSE.PROPRIETARY) we ship with packages now.

**Comment by tzach** [comment]
> What about our proprietary license we ship with packages now.

We should keep it, this is for the binary, and the new one is for the source.

File: README.md [hunk 1]
```diff
@@ -127,3 +127,7 @@ Before pushing please run `make check` and `make test` to make sure all tests pa
 
 When satisfied create a pull request onto master.
 All pull requests have to go trough review process before they are merged.
+
+## License
+
+Scylla Manager Source Code is available under Scylla Source Available License (SSAL) https://www.scylladb.com/scylla-source-available-license/
```
Comment (mmatczuk) [inline README.md:133]:
```suggestion
Scylla Manager Source Code is available under [Scylla Source Available License (SSAL)](https://www.scylladb.com/scylla-source-available-license/).
```

**Review by mmatczuk — APPROVED** [review]

**Pull request merged**